utils/log.go

```diff
@@ -91,10 +91,14 @@ func NewNopZapLogger() *ZapLogger {
 	return &ZapLogger{zap.NewNop().Sugar()}
 }
 
-func NewZapLogger(logLevel LogLevel) (*ZapLogger, error) {
+func NewZapLogger(logLevel LogLevel, color bool) (*ZapLogger, error) {
 	config := zap.NewProductionConfig()
 	config.Encoding = "console"
-	config.EncoderConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	if color {
+		config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+	} else {
+		config.EncoderConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	}
```

Comment:
Thanks for the contribution!
```suggestion
	config.EncoderConfig.EncodeLevel = zapcore.CapitalLevelEncoder
	if color {
		config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
	}
```